contrib/src/templates/mod.rs

```diff
@@ -214,27 +242,46 @@ impl Template {
     ///
     /// Returns `Some` if the template could be rendered. Otherwise, returns
     /// `None`. If rendering fails, error output is printed to the console.
+    /// `None` is also returned if a Template fairing has not been attached.
     ///
     /// # Example
     ///
     /// ```rust
     /// use std::collections::HashMap;
+    ///
+    /// extern crate rocket;
```

Comment:
Do we need this one? If so, move it to the top of the example and hide it using `# extern crate rocket`. Import the `Client` under the `use rocket_contrib::Template` line: `use rocket::local::Client`.